FakebookNotifications/FakebookNotifications.Testing/MongoDbTests.cs

```diff
@@ -0,0 +1,56 @@
+using FakebookNotifications.DataAccess;
+using FakebookNotifications.DataAccess.Models;
+using FakebookNotifications.DataAccess.Models.Interfaces;
+using Microsoft.Extensions.Options;
+using Moq;
+using System;
+using Xunit;
+
+namespace FakebookNotifications.Testing
+{
+    public class MongoDbTests
+    {
+        //Test if connection to database is successfull
+        [Fact]
+        public void MongoDbConnectionTest()
+        {
+            //Arrange
+            var mockSettings = Mock.Of<IOptions<NotificationsDatabaseSettings>>();
+            var mockContext = new Mock<NotificationsContext>(mockSettings.Value);
```

Comment:
this is misusing Mock. you have to have a very precise idea of what this test is testing. if it's the context's Connect method, then mocking the context makes zero sense. you mock the _dependencies_ of the test subject. this still does test the code, because mocking a class (as opposed to an interface) is using the real implementations by default, but it's semantically wrong.